user/how-to-guides/how-to-organize-your-qubes.md

```diff
@@ -0,0 +1,451 @@
+end up fine, but, she reminded herself, most people also don't have as much to
+lose. It's not a risk that she was willing to take with her future, especially
+knowing that there's probably no government bailout waiting for her and that
+all the brokerage firms' vaguely reassuring marketing language about
+cybersecurity isn't legally binding. So, Carol started reading more about
+computer security and eventually stumbled upon Qubes OS after searching the web
+for "most secure operating system." She read about how it's designed and why.
+Although she didn't immediately understand all of the technical details, the
+fundamental principle of [security-by-compartmentalization](/doc/architecture/)
+made intuitive sense to her, and the more she learned about the technical
+aspects, the more she realized that this is what she'd been looking for. Her
+setup looks like this:
+
+- One qube for each investment firm and bank. Carol has a few different
+  retirement accounts, brokerage accounts, and bank accounts. She treats each
+  qube like a "secure terminal" for accessing only that one institution's
+  website and saving any statements and confirmations she downloads in that
+  qube. She uses the [Qubes firewall](/doc/firewall/) to enable access only to
+  that institution's website so that she doesn't accidentally visit any others
+  in that qube.
+
+- One qube for all her credit card accounts. Carol considered making a separate
+  qube for each credit card account but ultimately decided against it. For one
+  thing, the consumer protections for credit card fraud in her country are much
+  better than for losing assets to theft or fraud in a bank or brokerage
+  account, so the security risk isn't as high. Second, there's actually not a
+  whole lot that an attacker could do with access to her credit cards' online
+  accounts or her old credit card statements, since online access to these
+  generally doesn't allow spending or withdrawing any money. So, even the worst
+  case scenario here wouldn't be catastrophic, unlike with her bank and
+  brokerage accounts. Finally, she has way too many credit cards! While she's
+  very frugal, she likes to collect the sign-up bonuses that are offered for
+  opening new cards, so she's accumulated quite a few of them. (However, she's
+  always careful to pay off her balance each month, so she never pays interest.
+  She's also pretty disciplined about only spending what she would have spent
+  *anyway* and not being tempted to spend more just to meet a spending
+  requirement or because she can.)
+
+- One qube for credit monitoring, credit reports, and credit history services.
+  Carol has worked hard to build up a good credit score, and she's concerned
+  about identity theft, so she has one qube dedicated to managing her free
+  credit monitoring services and downloading her free annual credit reports.
+
+- One qube for taxes. This is an offline qube where she stores all of her
```

Comment:
Maybe two qubes for taxes, one for keeping tax archives and one for filing them? I don't have that much of a universal experience, but all places I lived in over the last decade basically had online filing of taxes as the default.

Reply:
Very true! I didn't to get into that because I do my taxes on Windows :P, but actually this is a great opportunity to mention Windows qubes!